### DISCUSSION TITLE
Opensuse - unable to run "python3 -m pip install streamdeck-linux-gui --user" to install streamdeck-UI.

### DISCUSSION THREAD

**Discussion by fliptoback** [streamdeck-linux-gui/streamdeck-linux-gui#221] [Q&A]
Hi guys,

I am trying to install Streamdeck-UI on my tumbleweed with KDE plasma 6.

I followed the instructions on https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/main/docs/installation/opensuse.md

Steps I have done:
1) sudo zypper install libhidapi-libusb0 python312-devel kernel-devel python311-evdev
This went fine.

2) python3 -m pip install --upgrade pip
This gives me an error as follows:
```
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try
    zypper install python311-xyz, where xyz is the package
    you are trying to install.
    
     If you wish to install a non-rpm packaged Python package,
    create a virtual environment using python3.11 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip.
    
    If you wish to install a non-rpm packaged Python application,
    it may be easiest to use `pipx install xyz`, which will manage a
    virtual environment for you. Install pipx via `zypper install python311-pipx` .

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```
3) sudo wget https://raw.githubusercontent.com/streamdeck-linux-gui/streamdeck-linux-gui/main/udev/60-streamdeck.rules -O /etc/udev/rules.d/60-streamdeck.rules
This went fine

4) sudo udevadm trigger
This went fine

5) python3 -m pip install streamdeck-linux-gui --user
This again gives an error:

```
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try
    zypper install python311-xyz, where xyz is the package
    you are trying to install.
    
    If you wish to install a non-rpm packaged Python package,
    create a virtual environment using python3.11 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip.
    
    If you wish to install a non-rpm packaged Python application,
    it may be easiest to use `pipx install xyz`, which will manage a
    virtual environment for you. Install pipx via `zypper install python311-pipx` .

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

Not sure what I am doing wrong - appreciate some insight please.

Thanks in advance.

**Comment by tazmo** [comment]
Maybe try to install pipx as a system package, then use pipx to install streamdeck-linux-gui (the newer name) from pypi with:
```shell
pipx install streamdeck-linux-gui
```

**Reply by coolapso** [reply]
Reading the pinned discussion is probably wiser: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203

**Comment by coolapso** [comment]
Hi, streamdeck-linux-gui is on path to deprecation, please read: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203

**Comment by fliptoback** [comment]
Thanks Bob. I will give it a try. Appreciated the heads up.
